src/content/release/breaking-changes/android-predictive-back.md

```diff
@@ -12,6 +12,14 @@ To support Android 14's Predictive Back feature, a set of ahead-of-time APIs
 have replaced just-in-time navigation APIs, like `WillPopScope` and
 `Navigator.willPop`.
 
+:::note
+The Flutter 3.22 release includes some updates
+to predictive back behavior. For more info, check out
```

Comment:
**suggestion:** Avoid "For more..." construction. It lacks as a call to action and probably shouldn't abbreviate "info".

```suggestion
to predictive back behavior.
To learn more, check out
```